src/common/calendar/TimePickerCalendar.js

```diff
@@ -396,7 +396,7 @@ class TimePickerCalendar extends Component {
               click: () => onDateChange(this.calendarRef.current.calendar.view.initialNowDate),
             },
           }}
-          datesRender={this.onDatesRender}
+          // datesRender={this.onDatesRender}
```

Comment:
Any special reason for leaving this line commented out instead of deleted? In that case a comment to explain why it is still her is helpful!

Reply:
I actually just forgot to uncomment after debugging. I had a problem with an infinite loop. I fixed a similar issue for `resourcemap` so I attempted to see if that was the root of my issue and forgot this in. I don't think this prop does anything, but it's not within the scope of these changes to tamper with it so I'll just remove the comment for now.